src/stedi/cdk.clj

```diff
@@ -20,39 +20,6 @@
     (doseq [[fqn alias*] fqn+alias]
       (import/import-as-namespace fqn alias*))))
 
-(defmacro ^:deprecated require
-  "
-  Deprecated in favor of `stedi.cdk/import`
```

Comment:
Are the changes in this PR generally coupled to, or require, the removal of `require`?

Reply:
Yes, https://github.com/StediInc/cdk-clj/pull/43/files#diff-a8c46bb6da83b3863f99fbd3fa266650L46-L56 removes the ability to use the old pattern of invoking by keyword which the `require` API relied on.